tinygrad/tensor.py

```diff
@@ -216,6 +216,19 @@ def kaiming_normal(*shape, a:float = 0.01, **kwargs) -> Tensor:
     std = math.sqrt(2.0 / (1 + a ** 2)) / math.sqrt(prod(shape[1:]))
     return Tensor.normal(*shape, mean=0.0, std=std, **kwargs)
 
+  @staticmethod
```

Comment:
This shouldn't be a staticmethod, and the first arg should be self.

Fixes `p.multinomial` and is a line shorter